Projects/FeatureAuth/Scene/View/LoginButtonsView.swift

```diff
@@ -161,7 +161,7 @@ final class LoginButtonsView: BaseView {
 	override func setupView() { }
 }
 
-// MARK: - Action
+//MARK: - Actions
```

Comment:
```suggestion
// MARK: - Actions
```
수정하신 김에 이렇게 수정 부탁드립니다..! 😅